src/include/base/numbers.h

```diff
@@ -6,4 +6,5 @@
 int _getHexString(char*, unsigned long, bool);
 int _getDecString(char*, long);
 int _getUnsignedDecString(char *, unsigned long);
+int _itoa(char *, unsigned long, int, bool);
```

Comment:
this is mostly my fault not yours, but recentý i realized that not having variable names in header functions is not helpful to readability, so i started to add them whenever i can. But since i haven't touched the numbers.h in ages i haven't done it. So if you don't mind can you please add the names to the function header you created (and the others if you want, otherwise just your function is fine)